lib/matplotlib/quiver.py

```diff
@@ -87,21 +87,28 @@
 angles : {'uv', 'xy'} or array-like, default: 'uv'
     Method for determining the angle of the arrows.
 
-    - 'uv': Arrow direction in screen coordinates. Use this if the arrows
-      symbolize a quantity that is not based on *X*, *Y* data coordinates.
+    - 'uv': Arrow directions are based on screen coordinates (the coordinate
+       system of the plot, not the data). This is useful when the arrows represent 
```

Comment:
"coordinate system of the plot" is not a well-defined term.

```suggestion
    - 'uv': Arrow directions are based on
      :ref:`display coordinates <coordinate-systems>`; i.e. a 45° angle will
      always show up as diagonal on the screen, irrespective of figure or Axes
      aspect ratio or Axes data ranges. This is useful when the arrows represent 
```

Reply:
how and what should I implement in :ref:`display coordinates <coordinate-systems>;`? Please guide me.

Reply:
Nothing this is a reference to an existing target: https://matplotlib.org/devdocs/users/explain/artists/transforms_tutorial.html#coordinate-systems

Reply:
Got it, thank you.